src/LinearMap.jl

```diff
@@ -328,7 +329,7 @@ A list of vertices.
 
 ### Algorithm
 
-We assume that the underlying set `X` is polytopic.
+We assume that the underlying set `X` is polyhedral.
```

Comment:
what about using an interface for the dispatch? `LinearMap{N, S} where {S<:AbstractPolytope}`

Reply:
1. This PR is about `constraints_list`.
2. This is the old dispute about lazy operations. In this PR (`constraints_list`) I used the interface as you did in `Intersection`, but to be honest now things are inconsistent.